### PR TITLE
Edmundson Ch. 10

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/11-chapter-10.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-Of the just judgment of God upon George Clapham, the cruel priest.
-In 1682, William Edmundson was excommunicated,
-and had a public conference with the bishop, in his court, about tithes.
-Gospel ministers, worship, etc.
-Of his voyage to the West Indies, in 1683.
+[.chapter-synopsis]
+* The just judgment of God upon George Clapham, the cruel priest
+* In 1682, William Edmundson was excommunicated, and had a public conference with the bishop, in his court, about tithes
+* Gospel ministers, worship, etc.
```
Comment (Henderjay) [inline journal/modernized/11-chapter-10.adoc]: The words _Gospel ministers, worship, etc_. should follow the word _tithes,_ on the line above, as a continued list of subjects in their public conference. (Or is this an example of where you said the editor was making its own asterisk?)

File: journal/original/11-chapter-10.adoc [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+== Chatper X
 
-Of the just judgment of God upon George Clapham, the cruel priest.--In 1682,
-William Edmundson was excommunicated, and had a public conference with the bishop,
-in his court, about tithes.--Gospel ministers, worship,
-etc.--Of his voyage to the West Indies, in 1683.
+[.chapter-synopsis]
+* The just judgment of God upon George Clapham, the cruel priest
+* In 1682, William Edmundson was excommunicated, and had a public conference with the bishop, in his court, about tithes
+* Gospel ministers, worship, etc.
```
Comment (Henderjay) [inline journal/original/11-chapter-10.adoc]: Same thing here.

**Review by Henderjay — CHANGES_REQUESTED** [review]
Not sure if you meant to put _Gospel Ministers, worship, etc._ as its own synopsis line or if this is an example of where it created its own asterisk. ? Either way, lets bump that line up and connect it to the one above.👍🏻

File: journal/modernized/11-chapter-10.adoc (hunk 1 above)
Comment (maijapapaya) [reply journal/modernized/11-chapter-10.adoc]: This definitely could be an example, I really don't think I would have done that!!

**Comment by Henderjay** [comment]
Looks good, merging.

**Pull request merged**